packages/forms-web-app/src/dynamic-forms/validator/validator.js

```diff
@@ -11,12 +11,18 @@ const validate = () => {
 		const journeyResponse = res.locals.journeyResponse;
 		const journey = getJourney(journeyResponse);
 
-		const questionObj = journey.getQuestionBySectionAndName(section, question);
+		let questionObj = journey.getQuestionBySectionAndName(section, question);
 
 		if (!questionObj) {
 			throw new Error('unknown question type');
 		}
 
+		// add more question pages to validate subquestion if not currently adding a subquestion
+		const isAddMorePage = req.body['add-more-question'];
```

Comment:
wasn't happy with this but couldn't see a better way to know which validation to hit